modules/social_features/social_event/social_event.install

```diff
@@ -167,3 +179,31 @@ function social_event_update_8801() {
     $settings->set('enroll', [])->save();
   }
 }
+
+/**
+ * Remove public group from event.small_teaser.
+ */
+function social_event_update_8802() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_event', 'social_event_update_8802');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Update view mode for new small teaser style.
+ */
+function social_event_update_8803() {
```

Comment:
Probably these update helper update_hooks need to have a dependency on social_core 8805 which enables the update helper module otherwise we're not sure if these are fired in the correct order.

Reply:
We have dependency between previous update hooks (**social_event_update_8802** will be executed after **social_core_update_8802**) so it should implement your suggestion

Reply:
Yes, but update helper is only enabled in 8805, not 8802. That's where features is removed.
So the dependency for using update helper should be on 8805. 

Have you tried updating from a previous version? 

You will see everything is nicely waiting for 8802 but failing due to the fact 8805 hasn't run yet.